test/integration/run-test.sh

```diff
@@ -87,6 +88,9 @@ echo "unstage volume test:"
 "$CSC_BIN" node unstage --endpoint "$endpoint" --staging-target-path "$staging_target_path" "$volumeid"
 sleep 2
 
+# reset secrets as empty
+export X_CSI_SECRETS=""
+
```

Comment:
why reset secrets?

Reply:
otherwise DeleteVolume would fail since CreateVolume already failed, anyway, I filed an issue to track: https://github.com/kubernetes-csi/csi-driver-smb/issues/296